zh/configure-cluster-using-tidbcluster.md

```diff
@@ -82,3 +82,22 @@ spec:
       format: "format"
       disable-timestamp: false
 ```
+
+## 配置 TiFlash 配置参数
+
+你可以通过 TidbCluster Custom Resource 的 `spec.tiflash.config` 来配置 TiFlash 配置参数，以下是一个例子。
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  ...
+  tiflash:
+    config:
+      config:
```

Comment:
two `config`?

Reply:
Yes, there are two config structs under the `tiflash.config`.